### DISCUSSION TITLE
CssBaseline li:before perhaps not a good default?

### DISCUSSION THREAD

**Discussion by dpeek** [zeit-ui/react#475] [Ideas]
Given `ul > li` is frequently used for navigation rather than text content, having to override the `ul > li:before` was unexpected to a new comer. Perhaps some of these "prose" default styles could apply only inside `<Text>` components?

**Comment by unix** [comment]
The trade-offs here are indeed difficult to strike a balance, for some people, using `Text` everywhere is also very tedious. The truth is that we do lack a way to avoid style overrides.

Perhaps something like the `Pure` component is an option:

```tsx
// css baseline
li:not(.geist-pure) { ... }

// pure component
const Pure = () => (<><Children className="geist-pure" /></>)

// user case
<MyHeader>
  <Pure>
    <ul><li></li></ul>
  </Pure>
</MyHeader>
```

This is just a simple idea, is there a better way to create an isolated scope? (without affecting the user's styles)

(If you need this change very urgently, you can temporarily override it with your own CSS.)

**Reply by dpeek** [reply]
Thanks for the response. No urgency, you can still override once you understand that it is a `:before` and not just `list-style`. I would argue that the case where you want `ul > li` to have dashes is probably less common then where you don't... in that body content or markdown is usually injected into a specific location on a page and therefore easier to wrap in `Text` or something similar.

By no means something you have to change, only speaking from the perspective of a new user, once I've wrapped my head around the geist way it might make sense :) Thanks for the great library!
